.github/auto_assign.yml

```diff
@@ -0,0 +1,24 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+
+# Set to true to add assignees to pull requests
+addAssignees: true
```

Comment:
What do we want assignee to mean in the context of PRs? I think we're covered with author and reviewers

Reply:
I'm not sure @tjoshum. I've never used it, it might be that it is gone before we even know it =)